app/controllers/agent/manage.go

```diff
@@ -151,13 +153,24 @@ func addVisualExplain(data []byte) ([]byte, error) {
 	}
 	rawExplain := strings.Join(rawExplainRows, "\n")
 	rawExplain = strings.NewReplacer("<nil>", "NULL", "'", "").Replace(rawExplain)
+	rawExplain += "\n"
```

Comment:
Could you please add a comment here, that this newline is dramatically important :)